tests/manager/manager-executeBulkWrite-004.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-MongoDB\Driver\Manager::executeBulkWrite() with upserted ids
+MongoDB\Driver\Manager::executeBulkWrite() delete multiple documents
```

Comment:
`manager-executeBulkWrite-004.phpt` was essentially renamed to `manager-executeBulkWrite-002.phpt` after moving the duplicate key error tests. See https://github.com/mongodb-labs/mongo-php-driver-prototype/pull/128/files#r42785544.